internal/data/opentelemetry-proto-gen/collector/trace/v1/trace_service.pb.gw.go

```diff
@@ -59,7 +59,9 @@ func local_request_TraceService_Export_0(ctx context.Context, marshaler runtime.
 	if err := marshaler.NewDecoder(newReader()).Decode(&protoReq); err != nil && err != io.EOF {
 		return nil, metadata, status.Errorf(codes.InvalidArgument, "%v", err)
 	}
-
+	for key, val := range req.Header {
+		ctx = context.WithValue(ctx, key, val)
+	}
```

Comment:
Read the first few lines of the file:

```
// Code generated by protoc-gen-grpc-gateway. DO NOT EDIT.
// source: opentelemetry/proto/collector/trace/v1/trace_service.proto
```

Reply:
Thank you for review, I will study the code again and see how to correctly add the header to ctx

Reply:
Also, I don't see this file in our repo anymore.